Source/UZKFileInfo.m

```diff
@@ -105,4 +107,20 @@ - (NSDate *)readDate:(tm_unz)date
     return [calendar dateFromComponents:components];
 }
 
+- (BOOL)isDirectoryWith:(unz_file_info64 *)fileInfo {
+    
+    uLong type = fileInfo->external_fa >> 0x1D & 0x1F;
+    if (0 == (fileInfo->version >> 8)) {    //is DOS-archive
+        type = fileInfo->external_fa >> 4;
+        return (type == 0x01) && ![self isSymLinkWith:fileInfo->external_fa];
```

Comment:
- [ ] As in my comments above, I'd like this `0x01` to be defined in an enum along with the other possible values for this header field

Reply:
I need your help with the new type definition, which can reduce the number of times I am modifying the code. I will re-submit the new code in 2.0, after the type definition I need you to help me make the changes.